client-lib/go/client/jvs_client_test.go

```diff
@@ -109,32 +86,54 @@ func TestValidateJWT(t *testing.T) {
 	if err != nil {
 		t.Fatal("Couldn't get signing string.")
 	}
-	unsignedJWT := string(unsig)
+	unsignedJWT := string(unsig) + ".NOT_SIGNED"
 
 	split := strings.Split(validJWT2, ".")
 	sig := split[len(split)-1]
 
-	invalidSignatureJWT := unsignedJWT + sig // signature from a different JWT
+	invalidSignatureJWT := string(unsig) + sig // signature from a different JWT
+
+	breakglassTok := createBreakglassToken(t, "test_id_3")
+	breakglassTokStr, err := jwt.NewSerializer().Serialize(breakglassTok)
+	if err != nil {
+		t.Fatal("Couldn't get breakglass token string.")
+	}
+	breakglassJWT := string(breakglassTokStr) + ".NOT_SIGNED"
```

Comment:
What's the reason to _not_ wrap all lines here in the `createBreakglassToken` func?

Reply:
Followed the logic here. https://github.com/abcxyz/jvs/blob/3b5e51097c09a19a4f2ecfff5be9c6ccb1fcdc3e/client-lib/go/client/jvs_client_test.go#L169
The return of `createBreakglassToken` is a jwt token.

Reply:
And we need both type `token` and type `string`

Reply:
I've extracted such serialize logic into its own function to make it clear.